.commitlintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": ["@commitlint/config-conventional"]
+  "extends": ["@commitlint/config-conventional"],
+  "ignores": [
+    (commit) => commit.startsWith("Revert ")
```

Comment:
this looks like JS - should we rename the file to clarify and get better IDE integration?

Reply:
Yes, I can do that. Any other changes I need to make?

Reply:
Hi, I'm unable to import the types needed for this action. Do you have any idea why?

![commitlint](https://github.com/latticexyz/mud/assets/79749200/9b35e416-156f-4fcc-9d6b-517e1578ec28)

Reply:
@holic I would truly appreciate some help here.

Reply:
have you installed `@commitlint/types` as a new dependency?

Reply:
Hi @alvrs, when I tried to install the dependency before, I encountered an error message that went something like  `pnpm.ps1 cannot be loaded because running scripts is disabled on this system. For more information, see...`. I switched to a different IDE, and now I get an entirely different message. Am I not supposed to install the dependency in the root folder?

Reply:
I got it to work. Please review the changes.